apps/condo/domains/acquiring/routes/paymentLinkRouter.js

```diff
@@ -180,14 +195,34 @@ class PaymentLinkRouter {
                 )
             } else if (this.isInvoices(req)) {
                 const params = this.extractInvoicesParams(req)
-                await this.redirectIfAlreadyPaid(res, params, this.checkInvoicesAlreadyPaid.bind(this))
-
-                return await this.handlePaymentLink(
-                    res,
-                    params,
-                    this.invoicesPaymentValidator.bind(this),
-                    this.createMultiPaymentByInvoices.bind(this),
-                )
+                const isAlreadyPaid = await this.checkInvoicesAlreadyPaid(params)
+                if (isAlreadyPaid) {
+                    return this.redirectIfAlreadyPaid(res, params)
+                }
+
+                try {
+                    return await this.handlePaymentLink(
+                        res,
+                        params,
+                        this.invoicesPaymentValidator.bind(this),
+                        this.createMultiPaymentByInvoices.bind(this),
+                        true,
+                    )
+                } catch (error) {
+                    logger.error({
+                        msg: error.message,
+                        params: req.query,
+                        error,
+                    })
+
+                    const { failureUrl } = params
+                    const errorPageUrl = `${conf.SERVER_URL}/500-error.html`
```

Comment:
500-error is an old page
```suggestion
                    const errorPageUrl = `${conf.SERVER_URL}/500`
```